src/api/sessionAPI.js

```diff
@@ -0,0 +1,82 @@
+import axios from './axios';
+
+const LOGIN_URL = '/oauth/token';
+const SIGNUP_URL = '/users';
+const LOGOUT_URL = '/oauth/revoke';
+const CURRENT_USER_URL = '/users/me';
+
+export async function createUserWithEmailAndPassword(
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/231802512-c9cb32b6-fde1-43ee-8523-288479745375.png)

That way, your code will meet the latest standards.  👍